rusk/Cargo.toml

```diff
@@ -41,12 +41,14 @@ rustc_tools_util = "0.2"
 rand = "0.8"
 lazy_static = "1.4"
 rusk-profile = { path = "../rusk-profile" }
-rusk-vm = "0.6"
+rusk-vm = "0.8"
```

Comment:
I think `rusk-vm` is still at 0.7; in draft PR you can point to the `git` branch – it won't pass dusk-analyzer, but you can still use it for checking the tests.